cmd/shield/commands/access/rotate.go

```diff
@@ -0,0 +1,47 @@
+package access
+
+import (
+	"os"
+
+	"github.com/starkandwayne/goutils/ansi"
+	"github.com/starkandwayne/shield/api"
+	"github.com/starkandwayne/shield/cmd/shield/commands"
+	"github.com/starkandwayne/shield/cmd/shield/commands/internal"
+	"github.com/starkandwayne/shield/cmd/shield/log"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+//Rotate - Rotates the encryption database keys
```

Comment:
Can we call this `shield rekey-master`?  @thomasmmitchell what do you think?

Reply:
Its a bit more intuitive than `shield rotate` imo

Reply:
rekey is more consistent with `safe`, and I think rotate alone is a little too vague. SHIELD isn't for dealing with keys, so rotate alone is like... rotate what. `shield rekey` might be fine. `rekey-master` may be more verbose than needed, unless we're going to be rekeying other things which doesn't... make sense (?).

Reply:
I figured `rekey-master` was better because it's longer and more annoying to type.  My first inclination was `shield change-the-master-password-yes-that-one-the-super-important-one`